Add support for green LEDs to API

mdegat01 opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->

## Proposed change

Add APIs for changing Green LEDs per its DBus interface (https://github.com/home-assistant/os-agent/pull/164)

## Type of change

<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/1905
- Link to cli pull request: https://github.com/home-assistant/cli/pull/423

## Checklist

<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

pvizeli commented:
I guess @agners changed a bit how LED would work in future. The LED are now get restored from Supervisor on first start and the dbus command directly control the state. Like I understand it, the Yellow will get the same changes for LED. All other options still would need a reboot

mdegat01 commented:
@pvizeli ok I changed it. Supervisor now stores changes to LEDs on disk like the others. It then posts the expected value for LEDs on startup when we connect to the dbus interface for the board.

Although all boards have a different config only one can be in use at the same time so I had them all share `board.json`. The one potential consequence of this is if a user migrates to a different board it will keep options which share the same name (like `power_led` for yellow and green) and drop the rest. This seemed like a good thing for the most part (keep what we can) but if you forsee an unexpected consequence I can adjust to make sure that doesn't happen.

I updated yellow with this PR as well in anticipation of the change you said. I kept the `REBOOT_REQUIRED` issue there for now and only dropped it from green.
